pydoctor/templatewriter/pages/__init__.py

```diff
@@ -78,7 +170,7 @@ def namespace(self, obj: model.Documentable) -> Sequence[Union[Tag, str]]:
         while ob:
             if ob.documentation_location is model.DocLocation.OWN_PAGE:
                 if parts:
-                    parts.append('.')
+                    parts.extend(['.', tags.wbr])
```

Comment:
Is this necessary? Since the module names are links, those `<a>` tags are already points at which the browser can decide to wrap the string.